index.html

```diff
@@ -299,6 +299,18 @@ <h2>
 
   <p>This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the Architect theme by <a href="https://twitter.com/jasonlong">Jason Long</a>.</p>
 
+  <a href="https://gitter.im/sqlitebrowser/sqlitebrowser?utm_source=badge&amp;utm_medium=badge&amp;utm_campaign=pr-badge&amp;utm_content=badge">
```

Comment:
We should probably leave off the tracking variables (`utm_source`, etc). :smile:

Reply:
Should we also remove them from the README?

Reply:
Oh, didn't notice them there.  Yep, definitely. :smile:

Reply:
Done